Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -122,6 +122,49 @@ public static GoogleCredential FromJson(string json)
         }
 
         /// <summary>
+        /// Create a <see cref="GoogleCredential"/> directly from the provided access token.
+        /// The access token will not be automatically refreshed.
+        /// </summary>
+        /// <param name="accessToken">The access token to use within this credential.</param>
+        /// <param name="accessMethod">Optional. The <see cref="IAccessMethod"/> to use within this credential.
+        /// If <c>null</c>, will default to <see cref="BearerToken.AuthorizationHeaderAccessMethod"/>.</param>
+        /// <returns>A credential based on the provided access token.</returns>
+        public static GoogleCredential FromAccessToken(string accessToken, IAccessMethod accessMethod = null)
+        {
+            accessMethod = accessMethod ?? new BearerToken.AuthorizationHeaderAccessMethod();
+            return new GoogleCredential(new AccessTokenCredential(accessToken, accessMethod));
+        }
+
+        internal class AccessTokenCredential : ICredential, IHttpExecuteInterceptor
+        {
+            public AccessTokenCredential(string accessToken, IAccessMethod accessMethod)
+            {
+                _accessToken = accessToken;
+                _accessMethod = accessMethod;
+            }
+
+            private string _accessToken;
```

Comment:
Make these readonly? (I still prefer fields above the constructor, but hey... :)